File: 3302.cpp
#include<stdio.h>

int main(){
    
    int casoTeste, numero, i;
        i = 1;
        scanf("%d", &casoTeste);
        while(casoTeste>0){
            casoTeste--;
            scanf("%d",&numero);
            printf("resposta %d: %d\n",i ,numero);
            i++;
    
        }
    
    return 0;
}

File: 3302.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        int casoTeste, numero, i;
        i = 1;
        casoTeste = input.nextInt();
        while(casoTeste>0){
            casoTeste--;
            numero = input.nextInt();
            System.out.printf("resposta %d: %d\n",i ,numero);
            i++;
        }
 
    }
 
}
